test/ReactAI.test.ts

```diff
@@ -10,64 +12,110 @@ describe("ReactAI", () => {
     jest.resetModules();
   });
 
-  it("fails initialization without instrumentation key", () => {
-    const ReactAI = require("../src/ReactAI").default;
-    expect(() => ReactAI.initialize({ instrumentationKey: "" })).toThrow();
-    expect(() => ReactAI.initialize({})).toThrow();
-  });
-
   it("initializes correctly", () => {
-    const ReactAI = require("../src/ReactAI").default;
-    ReactAI.initialize({ instrumentationKey: IKEY });
-    expect(ReactAI.rootInstance).not.toBe(undefined);
-    expect(ReactAI.rootInstance.config.instrumentationKey).toBe(IKEY);
+    let reactAI = new ReactAI();
+    let appInsights = new ApplicationInsights({
+      config: {
+        instrumentationKey: IKEY,
+        extensions: [reactAI],
+        extensionConfig: {
+          "ApplicationInsightsReactUsage": { debug: false }
+        }
+      }
+    });
+    expect(reactAI).not.toBe(undefined);
+    expect(appInsights).not.toBe(undefined);
   });
 
   it("sets debug mode as expected", () => {
-    const ReactAI = require("../src/ReactAI").default;
-    ReactAI.initialize({ instrumentationKey: IKEY, debug: true });
-    expect(ReactAI.isDebugMode).toBe(true);
+
+    let reactAI = new ReactAI();
+    let appInsights = new ApplicationInsights({
+      config: {
+        instrumentationKey: IKEY,
+        extensions: [reactAI],
+        extensionConfig: {
+          "ApplicationInsightsReactUsage": { debug: true }
```

Comment:
```suggestion
          ReactAI.extensionIdentifier: { debug: true }
```

Reply:
`[ReactAI.extensionIdentifier]` - with square brackets, or the compiler complains

Reply:
Fixed all instances to use constants. Thanks for pointing this out.